pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1682,7 +1682,8 @@ public boolean isTopicNsOwnedByBroker(TopicName topicName) {
     public void cleanUnloadedTopicFromCache(NamespaceBundle serviceUnit) {
         for (String topic : topics.keys()) {
             TopicName topicName = TopicName.get(topic);
-            if (serviceUnit.includes(topicName)) {
+            if (getTopicReference(topic).isPresent() && serviceUnit.includes(topicName)) {
```

Comment:
it's better to swap the position of `getTopicReference(topic).isPresent()` and `serviceUnit.includes(topicName)` to avoid too many calls of `getTopicReference(topic).isPresent()`

Reply:
this comment was not addressed